README.md

```diff
@@ -1,66 +1,33 @@
-## Foundry
+# Multichain Token Contracts
 
-**Foundry is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust.**
+This repository contains smart contracts for implementing multichain token deployments using two different models: Hub and Spoke, and Burn and Mint.
 
-Foundry consists of:
+## Contracts
 
--   **Forge**: Ethereum testing framework (like Truffle, Hardhat and DappTools).
--   **Cast**: Swiss army knife for interacting with EVM smart contracts, sending transactions and getting chain data.
--   **Anvil**: Local Ethereum node, akin to Ganache, Hardhat Network.
--   **Chisel**: Fast, utilitarian, and verbose solidity REPL.
+### HubToken
 
-## Documentation
+`HubToken` is designed for the Hub and Spoke model. It represents the central token on the hub chain where the total supply is maintained.
 
-https://book.getfoundry.sh/
+### PeerToken
 
-## Usage
-
-### Build
-
-```shell
-$ forge build
-```
-
-### Test
-
-```shell
-$ forge test
-```
+`PeerToken` is a flexible contract that can be deployed as either a spoke token in the Hub and Spoke model or as a token in the Burn and Mint model.
 
-### Format
+## Read more about deployment models
 
-```shell
-$ forge fmt
-```
+[Deployment Models](https://docs.wormhole.com/wormhole/native-token-transfers/overview/deployment-models) 
 
-### Gas Snapshots
-
-```shell
-$ forge snapshot
-```
-
-### Anvil
-
-```shell
-$ anvil
-```
-
-### Deploy
+## Usage
 
-```shell
-$ forge script script/Counter.s.sol:CounterScript --rpc-url <your_rpc_url> --private-key <your_private_key>
-```
+1. Deploy `HubToken` on the main chain for the Hub and Spoke model.
+2. Deploy `PeerToken` on spoke chains for the Hub and Spoke model, or on all chains for the Burn and Mint model.
+3. Configure the minter address in `PeerToken` to control minting permissions.
```

Comment:
Would it be clearer to split up this usage section into "Hub and Spoke" and "Burn and Mint" usage? Technically in the "Burn and Mint" usage, you don't follow step (1)